hack/ci/testdata/seed.yaml

```diff
@@ -182,6 +182,7 @@ spec:
           images:
             centos: "CentOS 8 (2021-07-05)"
             ubuntu: "Ubuntu Focal 20.04 (2021-07-01)"
+            flatcar: "Flatcar Stable (2022-08-05)"
```

Comment:
Did you tests against this release version ?

Reply:
Yes. I think so. 

The change was committed 12 days ago.

This test ran last Thursday. https://public-prow.loodse.com/view/gs/prow-dev-public-data/pr-logs/pull/kubermatic_kubermatic/10700/pre-kubermatic-dualstack-e2e-openstack-cilium/1562767405480939520